### PR TITLE
Implemented fetching rockets, and displaying rockets as a list.

### PR DESCRIPTION
**Pull request by muneebulrehman**
- Implemented redux store with appropriate actions.
- Used the API to fetch rockets data.
- Each rocket has five properties, id, rocket_name, description, image, and reserved.
- Displayed each rocket on the page with required details.

### CODE REVIEW

**Review by souad988 — COMMENTED** [review]
Hi @muneebulrehman 
good job you did almost all the work 
i have only one comment that you can check below

File: src/redux/rockets/rockets.js [hunk 1]
```diff
@@ -35,7 +29,8 @@ export const fetchData = () => async (dispatch) => {
         id: el.id,
         name: el.rocket_name,
         description: el.description,
-        image: el.flickr_images[0]
+        image: el.flickr_images[0],
+        reserved: false
```
Comment (souad988) [inline src/redux/rockets/rockets.js:33]: i don't understand why do you call dispatch for each item i think it's better to pass the array and call dispatch once

Comment (muneebulrehman) [reply src/redux/rockets/rockets.js:33]: I did not get you , what you are saying is creating a new array and storing data in that and then passing that array in dispatch. Is that right?

Comment (souad988) [reply src/redux/rockets/rockets.js:33]: yes exactly create the array then call dispatch once

**Pull request merged**